main/git/APKBUILD

```diff
@@ -166,8 +166,16 @@ daemon() {
 		"$pkgdir"/$_gitcoredir/git-http-backend \
 		"$pkgdir"/$_gitcoredir/git-shell \
 		"$subpkgdir"/$_gitcoredir/
+}
+
+daemon_openrc() {
+	pkgdesc="Git protocol daemon (OpenRC init scripts)"
+	depends="git-daemon=${pkgver}-r${pkgrel} ${depends_openrc}"
+	install_if="openrc ${pkgname}-daemon=${pkgver}-r${pkgrel}"
+	replaces="git"
```

Comment:
Why would this replace git?!

Reply:
Let me be more precise. Why would git-daemon-openrc ever replace git?

git has all the command binaries and -openrc should only have conf and init data files.

Reply:
why would git-daemon replace git? :p I wasn't sure on this one, I wasn't quite sure on the logic in the other cases, as the same holds for git-daemon. So either this was intended, or it's wrong for most/all other cases as well; and if that is so, I'll happily add a patch to this series to remove the other cases

Reply:
I'd be happy to look over other cases too.

For now, please remove that replaces line.

Reply:
Ok; i'll remove it here, and add a patch removing it for the other cases as well.